docs/doxygen/overviews/eventhandling.h

```diff
@@ -242,7 +242,7 @@ Now let us describe the semantic differences:
 
     <li>
         As a slight extension of the above, the handlers can also be unbound at
-        any time with Unbind<>() (and maybe rebound later). Of course,
+        any time with Unbind<>() (and may be rebound later). Of course,
```

Comment:
This changes the meaning slightly. Originally it was meant to say that the handlers could, perhaps, be re-bound later if needed. I'm not sure if the new wording really makes it more clear.